nominees/automatic-classification-and-3d-segmentation-of-bone-tumor-from-patient-ct-images.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Automatic Classification and 3D Segmentation of Bone Tumor from  Patient CT Images",
+  "aliases": [
+    "AutoTumorDetect"
+  ],
+  "description": "Aim of this project is to use Artificial intelligence, specifically, deep learning methods to automate 3d bone tumor detection, segmentation, 3d reconstruction and classification of pelvic bone tumour types by applying deep learning Convolutional Neural Networks on volume datasets of variety pelvic bone tumours types in nifty (.nii) format and apply convolutional networks UNET (in segmentation) and DenseNet (in classification) to extract the prediction training Model. All that done using python language and predefined image processing , convolution and image-to-numerical arrays-transforms libraries, Pytorch and MONAI  (Medical Open Network for AI )",
+  "website": "https://github.com/biomedicalengineering/3D-Pelvic-Bone-cancer-segmentation-and-classification",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/biomedicalengineering/3D-Bone-cancer-segmentation-and-classification/blob/main/LICENSE.md"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "Increase society health quality , giving faster and simpler way for both patients and medical staff , accurate segmentation of tumor size , less complications for surgery and patients , coverage of more patients in short time",
+      "evidenceURL": "https://github.com/biomedicalengineering/3D-Pelvic-Bone-cancer-segmentation-and-classification"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "aimodel"
+  ],
+  "repositories": [
+    {
+      "name": "main",
+      "url": "https://github.com/biomedicalengineering/3D-Bone-cancer-segmentation-and-classification/blob/main/Pelvic_Bone_Tumour_segmentation_3d.ipynb"
```

Comment:
I think this should just be the link to the repository, not the specific AI model file.